lib/src/local_working_copy.rs

```diff
@@ -1452,16 +1444,10 @@ impl TreeState {
         })?;
 
         let matcher = self.sparse_matcher();
-        let copy_records = Default::default();
         let mut changed_file_states = Vec::new();
         let mut deleted_files = HashSet::new();
-        let mut diff_stream = old_tree.diff_stream(new_tree, matcher.as_ref(), &copy_records);
-        while let Some(TreeDiffEntry {
-            source: _, // TODO handle copy tracking
-            target: path,
-            value: diff,
-        }) = diff_stream.next().await
-        {
+        let mut diff_stream = old_tree.diff_stream(new_tree, matcher.as_ref());
```

Comment:
the TODO went away here. Not sure if you wanted to keep it or not

Reply:
I removed it because i don't think the working copy needs to care about copies. We *could* implement updating across a rename as a move in the file system, but it adds complexity and probably makes things slower in most cases.